spylib/store.py

```diff
@@ -167,6 +186,17 @@ async def execute_gql(self, query: str, variables: Dict[str, Any] = {}) -> Dict[
                 raise ShopifyThrottledError(
                     f'Store {self.name}: The Shopify API token is throttling. '
                 )
+            if OPERATION_NAME_REQUIRED_ERROR_MESSAGE in errorlist:
+                raise ShopifyCallInvalidError(
+                    f'Store {self.name}: Operation name was required for this query. \
+                        This likely means you have multiple queries within one call \
+                        and you must specify which to run.'
+                )
```

Comment:
Very nice message!
You can actually do that because it's all surrounded by brackets. I think it's nicer.
```suggestion
                raise ShopifyCallInvalidError(
                    f'Store {self.name}: Operation name was required for this query. '
                    'This likely means you have multiple queries within one call '
                    'and you must specify which to run.'
                )
```